### ISSUE TITLE
object property encoding error

### ISSUE THREAD

**Issue by jdalton** [meriyah/meriyah#224]
```js
/* Generated from: /test/language/expressions/assignment/member-expr-ident-name-break-escaped.js */
memberExprIdentNameBreakEscaped: {
  var obj = {};
  obj.bre\u0061k = 42;
  assert.sameValue(obj['break'], 42, 'property exists');
}
```
incorrectly throws:
Dot property must be an identifier

**Comment by jdalton** [comment]
Thank you @3cp!

**Comment by 3cp** [comment]
Released in v4.3.1.
